samples/snippets/test_sample_firewall.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import time
+import uuid
+
+
+import google.auth
+from google.cloud import compute_v1
+
+import pytest
+
+try:
```

Comment:
Why is there a try wrapped around the import statement?

Reply:
That was left over from when I was running tests locally and I had some importing issues. Fixed in next commit.